src/pip/_internal/basecommand.py

```diff
@@ -274,34 +270,38 @@ def populate_requirement_set(requirement_set, args, options, finder,
 
         for filename in options.constraints:
             for req_to_add in parse_requirements(
-                    filename,
-                    constraint=True, finder=finder, options=options,
-                    session=session, wheel_cache=wheel_cache):
+                filename,
+                constraint=True,
+                finder=finder,
+                options=options,
+                session=session,
+                wheel_cache=wheel_cache,
+            ):
```

Comment:
I don't think that blindly applying the "one argument a line" rule helps here, but if we're taking the view that blindly applying the rules is what we should do, then meh. It's not particularly *bad*, just takes up more vertical space than I'd like.